packages/gatsby-plugin-sitemap/README.md

```diff
@@ -21,12 +21,11 @@ plugins: [
 ```
 
 Above is the minimal configuration required to have it work. By default, the
-generated sitemap will include all of your site's pages except the dev 404 page
-(`/dev-404-page/`).
+generated sitemap will include all of your site's pages, except the ones you exclude.
 
-The [default query](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-sitemap/src/internals.js#L16)
-as well as any of the other `defaultOptions` [here](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-sitemap/src/internals.js#L15)
-can be overridden - for example:
+The `defaultOptions` [here](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-sitemap/src/internals.js#L15) can be overridden. By default, we exclude`/dev-404-page/`,`/404`&`/offline-plugin-app-shell-fallback/`.
```

Comment:
We should *always* exclude these — we never want them added — not just make them the `defaultOption`. That runs the risk that people will exclude some paths and start including these.

Reply:
Sorry I think I worded that poorly 😅 , or I'm misunderstanding.

The defaultOptions for excluding paths will _always_ have `/dev-404-page/`,`/404`&`/offline-plugin-app-shell-fallback/`. If the user adds some paths to exclude in the plugin options, those  will _also_ be excluded.

Reply:
Ah cool — yeah, rewording this would be great then. Just say "We always exclude these pages". "default" means it can be overridden.

Reply:
Alright, I reworded it to be more clear 👍